core/modules/comment/comment.module

```diff
@@ -1479,20 +1469,29 @@ function comment_config_create_validate(Config $staging_config, $all_changes) {
  * replied to. This prevents people from changing or revising their statements
  * based on the replies to their posts.
  *
- * @param $op
- *   The operation that is to be performed on the comment. Only 'edit' is
- *   recognized now.
- * @param $comment
- *   The comment object.
+ * @param string $op
+ *   The operation that is to be performed on the comment. Possible values are:
+ *   - "create"
+ *   - "update"
```

Comment:
Comment::access() also accepted "delete", "view", and "approve"(?)

Reply:
Yes. So approve is the odd one out here. None of the other modules have it, and entity_access also doesn't support it. Should I create custom access function for that one instead? It seems like a better approach if we want to be consistent. e.g. `function comment_approve_access()`

Reply:
docs updated in https://github.com/backdrop/backdrop/pull/2451